pkg/apiserver/filters/devops.go

```diff
@@ -33,8 +33,9 @@ import (
 
 // WithDevOpsAPIServer proxy request to DevOps service if requests path has the APIGroup with devops.kubesphere.io
 func WithDevOpsAPIServer(handler http.Handler, config *jenkins.Options, failed proxy.ErrorResponder) http.Handler {
-	if config.DevOpsServiceAddress == "" || config.K8sBearerToken == "" {
-		klog.V(6).Info("The DevOps service address or k8s token is empty, the proxy of DevOps server was not enabled")
+	if config.DevOpsServiceAddress == "" || config.DevOpsPluginServiceAddress == "" || config.K8sBearerToken == "" {
```

Comment:
Thanks for your time. But this is also not right. If `DevOpsServiceAddress` and `K8sBearerToken` is not empty. Then it can proxy `serverAsProxy(w, req, config.DevOpsServiceAddress, config.K8sBearerToken)`. But it cannot proxy the plugin service.

Reply:
Got your point.